ansible/roles/common/tasks/main.yml

```
@@ -156,9 +156,17 @@
       - kubectl
 
 # Instructions: https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#configure-cgroup-driver-used-by-kubelet-on-control-plane-node
+- name: Create blank file for kubelet
+  file:
+    path: /etc/default/kubelet
+    state: touch
+    owner: root
+    group: root
+    mode: 0644
```

```suggestion
- name: Create a firmware folder if it does not exist
  file:
    path: /boot/firmware
    state: directory
    mode: '0755'

- name: Create the config files if they do not exist
  file:
    path: "{{ item }}"
    state: touch
  with_items:
    - /boot/firmware/nobtcmd.txt
    - /boot/firmware/syscfg.txt
```

This would also mean you need to revert the kubelet change in the path for the `Enable required cgroup features` step.